static/js/compose_banner.ts

```diff
@@ -80,5 +80,13 @@ export function show_stream_does_not_exist_error(stream_name: string): void {
     const $compose_banner_area = $("#compose_banners");
     $compose_banner_area.append(new_row);
     hide_compose_spinner();
-    $("#stream_message_recipient_stream").trigger("focus").trigger("select");
+
+    // TODO: Replace with compose_actions.open_compose_stream_dropup() when it is converted to ts.
```

Comment:
Hmm, I thought we could call JS code from TS code, it just won't usefully type-check?

Reply:
I get this error:

```
tsc       | static/js/compose_banner.ts(6,34): error TS7016: Could not find a declaration file for module './compose_actions'. '/srv/zulip/static/js/compose_actions.js' implicitly has an 'any' type.
```

Reply:
There are escape hatches, one of which would be to create `static/js/compose_actions.d.ts`:

```ts
export function open_compose_stream_dropup(): void;
```